src/software/pipeline/main_sfmBootstraping.cpp

```diff
@@ -176,58 +122,8 @@ bool estimatePairAngle(const sfmData::SfMData& sfmData,
     return true;
 }
 
-double computeScore(const feature::FeaturesPerView& featuresPerView,
-                    const track::TracksMap& tracksMap,
-                    const std::vector<IndexT>& usedTracks,
-                    const IndexT viewId,
-                    const size_t maxLevel)
-{
-    const feature::MapFeaturesPerDesc& featuresPerDesc = featuresPerView.getFeaturesPerDesc(viewId);
-
-    std::vector<std::set<std::pair<unsigned int, unsigned int>>> uniques(maxLevel - 1);
-
-    for (auto trackId : usedTracks)
-    {
-        auto& track = tracksMap.at(trackId);
-
-        const feature::PointFeatures& features = featuresPerDesc.at(track.descType);
-
-        const IndexT featureId = track.featPerView.at(viewId).featureId;
-        const Vec2 pt = features[featureId].coords().cast<double>();
-
-        unsigned int ptx = (unsigned int)(pt.x());
-        unsigned int pty = (unsigned int)(pt.y());
-
-        for (unsigned int shift = 1; shift < maxLevel; shift++)
-        {
-            unsigned int lptx = ptx >> shift;
-            unsigned int lpty = pty >> shift;
-
-            uniques[shift - 1].insert(std::make_pair(lptx, lpty));
-        }
-    }
-
-    double sum = 0.0;
-    for (unsigned int shift = 1; shift < maxLevel; shift++)
-    {
-        int size = uniques[shift - 1].size();
-        if (size <= 1)
-        {
-            continue;
-        }
-
-        double w = pow(2.0, maxLevel - shift);
-        sum += w * double(size);
-    }
-
-    return sum;
-}
 
-bool buildSfmData(sfmData::SfMData& sfmData,
-                  const sfm::ReconstructedPair& pair,
-                  const track::TracksMap& tracksMap,
-                  const feature::FeaturesPerView& featuresPerView,
-                  const std::vector<IndexT>& usedTracks)
+bool buildSfmData(sfmData::SfMData & sfmData, const sfm::ReconstructedPair & pair, const track::TracksMap& tracksMap, const std::vector<std::size_t> & usedTracks)
```

Comment:
I don't understand what does "buildSfmData".
Could we add a sentence to explain?